app/views/api/v2/snapshots/main.json.rabl

```diff
@@ -7,5 +7,5 @@ extends 'api/v2/snapshots/base'
 attributes :description
 
 node(:created_at, &:create_time)
-node(:parent_id) { |snapshot| snapshot.parent.try(:id) }
-node(:children_ids) { |snapshot| snapshot.children.map(&:id) }
+node(:parent_id) { |snapshot| snapshot.try(:parent).try(:id) }
+node(:children_ids) { |snapshot| snapshot.try(:children).try(:map, &:id) }
```

Comment:
Should this be an array? Something like

`Array.wrap(snapshot&.children).map(&:id)`?

Reply:
Honestly i do not know. I guess your version would give an empty array instead of nil. Not sure, what was better.